index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+  <title>Jerry's Market Place</title>
+</head>
+<body>
+  <nav>
+    <ul>
+      <li>HOME</li>
+      <li>PRODUCTS</li>
+      <li>CONTACT</li>
+      <li>ABOUT</li>
+    </ul>
+  </nav>
+
+  <div>
+    <h1 class="bjmarket">Benjamin & Geraldo's Frozen Treat Market Place
+    </h1>
+      <img class="headshot" src="http://www.mikethefanboy.com/wp/wp-content/uploads/2013/08/ben-affleck-e1377289867576.jpg" alt="Ben Affleck's Smiling Face">
+      <img class="headshot" src="http://mmbiz.qpic.cn/mmbiz/jAq9wqyDucxdfPicdfVUclXALrGph9DmviafggJia1fwxK2Yc9w88ekABZhu3ELpHFlANl6lfmlR4PObxuqq69zJw/640?/1.jpg" alt="Geraldo Rivera's Smiling Face">
+  </div>
+
+  <article class="icecream">
+    <div>
+    <section>
+      <header>
+        <h1>Cherry Garcia</h1>
+      </header>
+        <img class="pint" src="http://www.benjerry.com/files/live/sites/systemsite/files/flavors/products/us/pint/cherry-garcia-detail.png" alt "Ben and Jerry's Cherry Garcia">
```

Comment:
`<img>` tags are sort of the platypus of elements. They take on properties of both inline and block tags. To make sure you have a block level element try wrapping your image in a `<div>` tag.